framework/xml_helper.rb

```diff
@@ -15,24 +15,42 @@ class XmlParams
     odp: '131'
   }.freeze
 
+  # To convert some files via xml use a encoding decimal number
+  # The encoding numbers are in
+  # https://github.com/ONLYOFFICE/server/blob/af00c97fb63b609e699259ab23bf52aeb76fa256/Common/sources/commondefines.js#L951
+  ENCODING_NUMBERS = {
+    'UTF-8': '46',
+    '': ''
+  }.freeze
+
   # :fonts_path  - is a path to folder with fonts
   # :tmp_path  - is a path to temp folder
   def initialize(options = {})
     @fonts_path = options[:fonts_path]
     @tmp_path = options[:tmp_path]
   end
 
+  # @param [String] encoding_name encoding name
+  # @return [String] decimal encoding number
+  def encode_number_by_name(encoding_name)
+    raise "Unknown encoding: #{encoding_name}" if ENCODING_NUMBERS[encoding_name.to_sym].nil?
```

Comment:
I think it can be replaced with 
```
if ENCODING_NUMBERS.keys.include?(encoding_name)
```

Also please note that in current code `to_sym` is not needed to be done